event-handler/teleport_events_watcher.go

```diff
@@ -38,8 +39,14 @@ const (
 
 // TeleportSearchEventsClient is an interface for client.Client, required for testing
 type TeleportSearchEventsClient interface {
+	// SearchEvents searches for events in the audit log and returns them using their protobuf representation.
 	SearchEvents(ctx context.Context, fromUTC, toUTC time.Time, namespace string, eventTypes []string, limit int, order types.EventOrder, startKey string) ([]events.AuditEvent, string, error)
+	// StreamSessionEvents returns session events stream for a given session ID using their protobuf representation.
 	StreamSessionEvents(ctx context.Context, sessionID string, startIndex int64) (chan events.AuditEvent, chan error)
```

Comment:
Do we still use those?

Reply:
I left them if someone wants to write a custom exporter based on protobuf structures

Reply:
I think we should remove it, but not blocking because of it
In the future™️, someone will ask why we have those and will probably remove them :joy: